.env.example

```
@@ -34,9 +34,19 @@ RESET_TOKEN_SECRET=change-me-to-another-random-string
+QUEUE_DEFAULT_EVENT_CAP=10      # Cap fallback khi Worker chưa tính xong (lần đầu tiên)
+QUEUE_MAX_EVENT_CAP=200         # Trần cứng — không event nào được quá số này
+QUEUE_DYNAMIC_CAP_RATIO=0.1    # Tỉ lệ ghế còn lại: cap = ceil(remaining * 0.1)
+
+# Override (để demo Virtual Queue)
+# Cấu hình tối ưu để demo (số 3 là đẹp nhất)
+# QUEUE_DEFAULT_EVENT_CAP=3     # Ban đầu cho 3 người vào
+# QUEUE_MAX_EVENT_CAP=3         # Giới hạn tối đa cũng chỉ là 3 (để dễ demo hàng chờ)
+# QUEUE_DYNAMIC_CAP_RATIO=0.1    # Giữ nguyên tỉ lệ để code chạy đúng logic
+
+# Email
+RESEND_API_KEY="re_dummy_key_123456789"
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The RESEND_API_KEY key is duplicated

(DuplicatedKey)